configs/data/contrastive/vicreg.yaml

```diff
@@ -0,0 +1,136 @@
+_aux:
+  batch_size: 128
+
+train_dataloaders:
+  _target_: monai.data.dataloader.DataLoader
+  batch_size: ${data._aux.batch_size}
+  num_workers: 8
+  shuffle: true
+  persistent_workers: false
+  dataset:
+    _target_: cyto_dl.datamodules.torchvision.MNIST
```

Comment:
maybe relevant for future - AICS "MNIST" - https://open.quiltdata.com/b/allencell/tree/aics/aics_mnist/